build.sbt

```diff
@@ -9,16 +9,14 @@ ThisBuild / githubWorkflowBuildPreamble ++= List(
   WorkflowStep.Sbt(List("scalafmtCheckAll"), name = Some("Check formatting"))
 )
 
-lazy val publishSettings = Seq(
-  releaseCrossBuild := true,
-  UpdateVersionInFiles("sphere-util", file("README.md")),
-  UpdateVersionInFiles("sphere-json", file("README.md"), file("json/README.md")),
-  UpdateVersionInFiles("sphere-mongo", file("README.md"), file("mongo/README.md"))
-) ++ BintrayPlugin.bintraySettings
-
-lazy val standardSettings = Defaults.coreDefaultSettings ++ publishSettings ++ Seq(
-  organization := "io.sphere",
+inThisBuild(List(
+  organization := "com.commercetools",
   licenses += ("Apache-2.0", url("https://www.apache.org/licenses/LICENSE-2.0.html")),
+  homepage := Some(url("https://github.com/commercetools/sphere-scala-libs")),
+  developers := List(Developer(id = "commercetools", name = "commercetools", email = "ondemand@commercetools.com", url = url("https://commercetools.com")))
```

Comment:
`ondemand@commercetools.com` is the mail address in the imprint on https://commercetools.com. I did not want to put an internal address here because of spam and security concerns. I assume people would reach out via GitHub issues / PRs anyway.